src/ducks/authIntoSystem/authIntoSystemActions.jsx

```diff
@@ -0,0 +1,16 @@
+export const AUTH_SIGN_UP = "AUTH_SIGN_UP";
+export const AUTH_LOGIN = "AUTH_LOGIN";
+
+export const userSignUp = (data) => {
```

Comment:
if the action creator name below if user**Login**, this should be named in the same approach - userRegister. Or rename userLogin to userSingIn